azure/iac/aks.rego

```diff
@@ -442,23 +442,21 @@ azure_issue["aks_kub_dashboard_disabled"] {
     resource.properties.addonProfiles.kubeDashboard.enabled != false
 }
 
+
 aks_kub_dashboard_disabled {
-    lower(input.resources[_].type) == "microsoft.containerservice/managedclusters"
-    not azure_attribute_absence["aks_kub_dashboard_disabled"]
-    not azure_issue["aks_kub_dashboard_disabled"]
+    azure_attribute_absence["aks_kub_dashboard_disabled"]
```

Comment:
@shahinshirvani **what was the issue with existing rules?** kube dashboard is enabled by default if `kubeDashboard `does not exist. previous rules is correct. please revert if no significant issue found.

Reply:
@shahinshirvani turns out now dashboard does not get installed automatically.

**Change the rule like this one. current one is throwing error with multiple resource:**

```
default aks_kub_dashboard_disabled = null
azure_attribute_absence["aks_kub_dashboard_disabled"] {
    resource := input.resources[_]
    lower(resource.type) == "microsoft.containerservice/managedclusters"
    not resource.properties.addonProfiles.kubeDashboard.enabled
}
azure_issue["aks_kub_dashboard_disabled"] {
    resource := input.resources[_]
    lower(resource.type) == "microsoft.containerservice/managedclusters"
    resource.properties.addonProfiles.kubeDashboard.enabled != false
}

aks_kub_dashboard_disabled {
    azure_attribute_absence["aks_kub_dashboard_disabled"]
    not azure_issue["aks_kub_dashboard_disabled"]
}

aks_kub_dashboard_disabled {
    not azure_attribute_absence["aks_kub_dashboard_disabled"]
    not azure_issue["aks_kub_dashboard_disabled"]
}

aks_kub_dashboard_disabled = false {
    azure_issue["aks_kub_dashboard_disabled"]
}

aks_kub_dashboard_disabled_err = "Kubernetes Dashboard is currently not disabled" {
    azure_issue["aks_kub_dashboard_disabled"]
}

aks_kub_dashboard_disabled_metadata := {
    "Policy Code": "PR-AZR-0144-ARM",
    "Type": "IaC",
    "Product": "AZR",
    "Language": "ARM template",
    "Policy Title": "Ensure Kubernetes Dashboard is disabled",
    "Policy Description": "Disable the Kubernetes dashboard on Azure Kubernetes Service",
    "Resource Type": "microsoft.containerservice/managedclusters",
    "Policy Help URL": "",
    "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.containerservice/managedclusters"
}
```

**Test with below json:**

```
{
    "resources": [
        {
        	"type" : "microsoft.containerservice/managedclusters",
            "properties" : {
                "addonProfiles" : {
                    "kubeDashboard" : {
                        "enabled" : true
                    }
                }
            }
        },
        {
        	"type" : "microsoft.containerservice/managedclusters",
            "properties" : {
                "addonProfiles" : {
                    "kubeDashboard" : {
                        "enabled" : false
                    }
                }
            }
        }
    ]
}
```

Reply:
@rezoan so kubeDashboard is disabled by default
so I just updated rego file according to your comment